src/z3c/form/browser/checkbox.rst

```diff
@@ -1,6 +1,5 @@
-==============
-CheckBoxWidget
-==============
+CheckBox Widget
```

Comment:
If the title refers to the class name it should stay "CheckBoxWidget".  If it refers to a checkbox widget, it should be "Checkbox Widget".  BiCapitalization looks wrong to me.